### DISCUSSION TITLE
request to add -> expression of images in base64

### DISCUSSION THREAD

**Discussion by OcelusPRO** [Tyrrrz/DiscordChatExporter#1190] [Q&A]
Hey,
As part of the development of a discord bot which will manage tickets (as ticket-tools would do) I would like to export the ticket rooms, these rooms are deleted immediately after being logged.

The problem here is that I couldn't simply share an HTML file and have to send a zip with the images, so it would be interesting to be able to directly load the images in the HTML file in base64 format to only have this file shared when the room with the images was deleted.

Another discussion of this kind existed a few years ago, but the answer given was to open the file in the browser then save the page via the browser, which in my use case is not possible because the images are deleted before a user can do this operation

**Comment by Tyrrrz** [comment]
You can export with "download assets" option enabled to persist all referenced assets.
